configuration.go

```diff
@@ -12,6 +12,10 @@ type Configuration struct {
 	// ICE, such as STUN and TURN servers.
 	ICEServers []ICEServer
 
+	// IgnoreIPv6 is an optional configuration for disabling support for ipv6
+	// server reflexive candidates. Default is false.
+	IgnoreIPv6 bool
```

Comment:
I think we're putting non-standard options in SettingEngine instead of directly in these structs. Maybe move this there?

Reply:
Can you show me? A link.

Reply:
https://github.com/pions/webrtc/blob/master/settingengine.go#L12

Reply:
Thank you. I will try moving it there but it might be challenging since the only way for a user to pass in configurations is via `type Configuration struct{}`. Unless there is another way I am not thinking about.

Reply:
Okay so I just looked at it an unfortunately there is not way to avoid not putting this at least in `type Configuration struct{}`. I am going to add a deprecation comment to discourage people from using it.

Reply:
ICEGatherer is constructed by API and API has a SettingEngine right? A little convoluted but you can get the settings there.

Let's see what @Sean-Der thinks. He has stronger opinions about keeping `struct Configuration` standard than I do.

Reply:
Yeah. He actually was the one that said for now to just add the opt-out config. I'll wait for him to review.

Personally I am okay with completely removing ipv6 support right now but I know others use this library in other ways so I am taking a more "backward compatible kinda solution".

Reply:
One other thing I should probably do is make that variable local. This way it will not show up in the documentations at all and can be used only by people reading through the source code.

I'll just go ahead and do it.